pom.xml

```diff
@@ -49,6 +49,11 @@
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>display-url-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
This won't build against a snapshot in https://jenkins.ci.cloudbees.com - is there a release yet for display-url-api?